<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_books</class>
 <widget class="QDialog" name="show_books">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/images/bg_login2.jpg&quot;);
background-position: center</string>
  </property>
  <widget class="QPushButton" name="btn_voltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 82, 223);
font: 10pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="texto_frase">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,79,0);
font: 9pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string> &quot;A companhia dos livros dispensa com grande
vantagem a dos homens.&quot; - Marquês de Maricá</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="txt_inicio">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 14pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Estes são seus livros:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="txt_resultado">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 0);
font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Título
Autor:
Número de páginas totais:
Número de páginas lidas
Status:
Progresso:
Progresso2:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progresso_leitura">
      <property name="styleSheet">
       <string notr="true">background: rgb(0, 79, 0);
QProgressBar::chunk {
          background-color: #3498db;
        width: 20px;
};  </string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_proximo">
      <property name="styleSheet">
       <string notr="true">background: rgb(0, 79, 0);
color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Próximo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
